### DISCUSSION TITLE
I can't sign in https://www.npmjs.com/，when i log in,will show 404 route not found

### DISCUSSION THREAD

**Discussion by wangrongding** [npm/feedback#701] [Website]
I can't sign in https://www.npmjs.com/
my account: wangrongding

![115ebc3d39e2cd6c9a27072d0c008fa](https://user-images.githubusercontent.com/42437658/172037737-ffa5fbce-53d5-4641-a583-c4f1d5e1c367.png)

but, i can still use it.just can't log in.  

![7e2cb76ae8ad1e0318322a682ef5a82](https://user-images.githubusercontent.com/42437658/172037753-8c5af49a-6303-4c4a-91b8-4f2ad1e293b7.png)

**Comment by wangrongding** [comment]
I can't comment on the feedback page either.  


![1654410872860](https://user-images.githubusercontent.com/42437658/172038579-5151cd63-c9cd-41d5-8295-26ae183e863c.jpg)

**Comment by wangrongding** [comment]
I don't know if the government blocked it or if my account is disabled in npmjs.
I can only use it to find, but can't log in, I am troubled by this, I hope I can get help, thanks💗

**Comment by monishcm** [comment]
I believe your browser is doing the translation here causing issues while submitting. Recommend trying out the same steps with the English version - provided out of the box.

**Reply by wangrongding** [reply]
> I believe your browser is doing the translation here causing issues while submitting. Recommend trying out the same steps with the English version - provided out of the box.

![image](https://user-images.githubusercontent.com/42437658/172301627-f2c5d496-ff09-442c-94a6-f45169ae6812.png)

no, i tried😢

**Reply by wangrongding** [reply]
Also, this only happens in Chrome.
When I use Edge, it works fine

**Reply by wangrongding** [reply]
Use edge  to login : successful
Use Chrome to login : failed

Is it because npm was acquired by Microsoft? 😢
